### DISCUSSION TITLE
A new Idea

### DISCUSSION THREAD

**Discussion by 1190201123** [antlr/intellij-plugin-v4#575] [Ideas]
HI, I am a student from China. I want to try something new like a plugin to converting from a language to another based on antlr4 plugin. Is that permitted? PLEASE🥹🥹

**Comment by parrt** [comment]
hi!  ANTLR is a parser generator that allows you to part one language and provide support for you to translate to another but it is a very complicated task in general. The plug-in of course will only allow you to edit your grammars and is not directly help you with the translator.

**Reply by 1190201123** [reply]
I'm excited for your reply! You are right. I will learn more from ANTLR. Hoping i can make some contribution and be as talented as you!
